cmd/dmverity-vhd/main.go

```diff
@@ -72,16 +74,26 @@ func main() {
 
 func fetchImageLayers(ctx *cli.Context) (layers []v1.Layer, err error) {
 	image := ctx.String(imageFlag)
+	tarballPath := ctx.String(tarballFlag)
```

Comment:
It feels weird that `fetchImageLayers`, which is used in both `createVHDCommand` and `rootHashVHDCommand` checks for the `tarball` flag, when only `rootHashVHDCommand` declares it as a flag.
Does it make sense to allow the `tarball` flag to the create command as well?

Reply:
sounds good to me.